.github/ISSUE_TEMPLATE/bug-fix.md

```diff
@@ -0,0 +1,38 @@
+---
+name: Bug fix
```

Comment:
Do we need a separate category for bug fix, if you have a bug, you can add a section: work around 

if you have a real solution, it would be a pull request.

Reply:
> if you have a real solution, it would be a pull request.

That's true, I will remove it

Reply:
Did it in https://github.com/inmanta/inmanta/pull/2416/commits/5090b071922f322cee876f9d6288b7279514d3b2